CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   - `azureauth ado` : Prints the help for Azure Devops commands.
   - `azureauth ado pat` : Command for creating, and locally caching Azure Devops <abbr title="Personal Access Tokens">PAT</abbr>s.
   - `azureauth ado token` : Command for passing back a <abbr title="Personal Access Tokens">PAT</abbr> from an env var, or authenticating and returning an <abbr title="Azure Active Directory">AAD</abbr> access token.
+- Now supports reading data from the registry when the environment variable is not configured.
```

Comment:
This changelog entry needs to be more specific. It is not clear that the thing being read is the telemetry key, and that it's a back up/alternative to the `AZUREAUTH_APPLICATION_INSIGHTS_TOKEN` env var.